src/traci-server/TraCIConstants.h

```diff
@@ -483,495 +498,498 @@
-#define VAR_POSITION3D 0x39
+TRACI_CONST ubyte_t VAR_POSITION3D = 0x39;
 
 // angle (get: vehicle, poi; set: poi)
-#define VAR_ANGLE 0x43
+TRACI_CONST ubyte_t VAR_ANGLE = 0x43;
 
 // angle (get: vehicle types, lanes, arealdetector, set: lanes)
-#define VAR_LENGTH 0x44
+TRACI_CONST ubyte_t VAR_LENGTH = 0x44;
 
 // color (get: vehicles, vehicle types, polygons, pois)
-#define VAR_COLOR 0x45
+TRACI_CONST ubyte_t VAR_COLOR = 0x45;
 
 // max. acceleration (get: vehicles, vehicle types)
-#define VAR_ACCEL 0x46
+TRACI_CONST ubyte_t VAR_ACCEL = 0x46;
 
 // max. comfortable deceleration (get: vehicles, vehicle types)
-#define VAR_DECEL 0x47
+TRACI_CONST ubyte_t VAR_DECEL = 0x47;
 
 // max. (physically possible) deceleration (get: vehicles, vehicle types)
-#define VAR_EMERGENCY_DECEL 0x7b
+TRACI_CONST ubyte_t VAR_EMERGENCY_DECEL = 0x7b;
 
 // apparent deceleration (get: vehicles, vehicle types)
-#define VAR_APPARENT_DECEL 0x7c
+TRACI_CONST ubyte_t VAR_APPARENT_DECEL = 0x7c;
 
 // action step length (get: vehicles, vehicle types)
-#define VAR_ACTIONSTEPLENGTH 0x7d
+TRACI_CONST ubyte_t VAR_ACTIONSTEPLENGTH = 0x7d;
 
 // last action time (get: vehicles)
-#define VAR_LASTACTIONTIME 0x7f
+TRACI_CONST ubyte_t VAR_LASTACTIONTIME = 0x7f;
 
 // driver's desired headway (get: vehicle types)
-#define VAR_TAU 0x48
+TRACI_CONST ubyte_t VAR_TAU = 0x48;
 
 // vehicle class (get: vehicle types)
-#define VAR_VEHICLECLASS 0x49
+TRACI_CONST ubyte_t VAR_VEHICLECLASS = 0x49;
 
 // emission class (get: vehicle types)
-#define VAR_EMISSIONCLASS 0x4a
+TRACI_CONST ubyte_t VAR_EMISSIONCLASS = 0x4a;
 
 // shape class (get: vehicle types)
-#define VAR_SHAPECLASS 0x4b
+TRACI_CONST ubyte_t VAR_SHAPECLASS = 0x4b;
 
 // minimum gap (get: vehicle types)
-#define VAR_MINGAP 0x4c
+TRACI_CONST ubyte_t VAR_MINGAP = 0x4c;
 
 // width (get: vehicle types, lanes, polygons, poi)
-#define VAR_WIDTH 0x4d
+TRACI_CONST ubyte_t VAR_WIDTH = 0x4d;
 
 // shape (get: polygons)
-#define VAR_SHAPE 0x4e
+TRACI_CONST ubyte_t VAR_SHAPE = 0x4e;
 
 // type id (get: vehicles, polygons, pois)
-#define VAR_TYPE 0x4f
+TRACI_CONST ubyte_t VAR_TYPE = 0x4f;
 
 // road id (get: vehicles)
-#define VAR_ROAD_ID 0x50
+TRACI_CONST ubyte_t VAR_ROAD_ID = 0x50;
 
 // lane id (get: vehicles, inductionloop, arealdetector)
-#define VAR_LANE_ID 0x51
+TRACI_CONST ubyte_t VAR_LANE_ID = 0x51;
 
 // lane index (get: vehicle, edge)
-#define VAR_LANE_INDEX 0x52
+TRACI_CONST ubyte_t VAR_LANE_INDEX = 0x52;
 
 // route id (get & set: vehicles)
-#define VAR_ROUTE_ID 0x53
+TRACI_CONST ubyte_t VAR_ROUTE_ID = 0x53;
 
 // edges (get: routes, vehicles)
-#define VAR_EDGES 0x54
+TRACI_CONST ubyte_t VAR_EDGES = 0x54;
 
 // update bestLanes (set: vehicle)
-#define VAR_UPDATE_BESTLANES 0x6a
+TRACI_CONST ubyte_t VAR_UPDATE_BESTLANES = 0x6a;
 
 // filled? (get: polygons)
-#define VAR_FILL 0x55
+TRACI_CONST ubyte_t VAR_FILL = 0x55;
 
 // get/set image file (poi, poly, vehicle, person, simulation)
 #define VAR_IMAGFILE 0x93
```

Comment:
Why is this missing?

Reply:
Oops, that's simply a mistake. I will fix this!